crates/primitives/src/storage.rs

```diff
@@ -30,3 +30,33 @@ impl Compact for StorageEntry {
         (Self { key, value }, out)
     }
 }
+
+/// Account storage trie node.
+#[derive(Debug, Default, Clone, PartialEq, Eq, Serialize)]
+pub struct StorageTrieEntry {
+    /// Hashed storage key.
+    pub hash: H256,
+    /// Encoded node.
+    pub node: Vec<u8>,
+}
+
+// NOTE: Removing main_codec and manually encode subkey
```

Comment:
Can we still add `#[add_arbitrary_tests(comapct)]` ?

Reply:
Added!